src/subcommand/config.example.yml

```diff
@@ -0,0 +1,158 @@
+# This config file is read whenever broker starts, and contains all of the information that broker needs in order to work.
+#
+# To get up and running, follow the instructions in this file.
+#
+# There are two things you will absolutely need to change: the `fossa_integration_key` and the `integrations`.
+# Everything else can probably be left as is.
+#
+# By default there will be two .yml files in this directory.
+# config.example.yml is an example file that will not be used by broker, but can be used as a reference.
+# You can re-generate config.example.yml at any time by running `broker init`.
+
+# ==========
+
+# fossa_endpoint sets the endpoint that broker will send requests to.
+# This field should only be modified if your FOSSA account lives on a different server than app.fossa.com.
+# This is most commonly needed with on-premise instances of FOSSA.
+# In most cases this will be https://app.fossa.com.
+fossa_endpoint: https://app.fossa.com
+
+# fossa_api_key is the API key for your FOSSA account.
+# You can obtain a fossa API key at https://app.fossa.com/account/settings/integrations/api_tokens.
+# A push-only token will suffice, but you can use a full token as well if you wish.
+fossa_integration_key: abcd1234
+
+# version is the version of the config file format. "1" is the only currently supported version.
+version: 1
+
+# debugging configures where debug logs are written and how long they are kept for. The debugging section can probably remain as is.
+debugging:
+  # The location where your debug traces are written to. This can probably remain as is,
+  # but you can change it to a different directory if you would like to write them to a different location.
+  # This directory must exist and be writable.
+  location: {debugging_dir}
+  # How many days we retain the debug trace files for. The suggested default is 7 days.
+  retention:
+    days: 7
```

Comment:
Following on to the above comment, here's how I'd write this to be more helpful to the user.
The theme here is "explain what and why". If we want to empower them to be able to effectively choose new config values, we must explain what they're used for and why we chose the default we did.

```
# Configures where debug logs are written and for how long they are kept.
# We've configured suggested defaults for the debugging section below, see each key for details.
debugging:
  # Broker stores debug information, such as its debug log file and FOSSA CLI debug bundles,
  # inside this directory. We've configured the suggested default below based on the data root
  # provided to Broker at the time this file was created.
  # These files are only ever needed if you're reporting an issue with Broker to FOSSA.
  # If you do provide a different path, note that the path specified must be a directory and the user 
  # running Broker must have access to create files and subdirectories in that directory.
  location: {debugging_dir}

  # Broker cleans up debug information in the `location` provided above based on the time frame provided here.
  # The default is 7 days. We chose this because it should give users plenty of time to report an issue to FOSSA
  # and copy the debugging information out of the directory before it is cleaned up, without storing too much data.
  # The minimum retention is 1 day, and there is no maximum.
  retention:
    days: 7
```